utils/retrieve-state/src/main.rs

```diff
@@ -79,14 +83,26 @@ struct Opts {
     use_gzip: bool,
 
     #[structopt(
-        short,
+        short = "b",
         long,
         about = "Specify the path to the folder to be used for loading block files to be converted."
```

Comment:
I thought we were using LMDB now?

It might be a little easier if we just had a `--data-dir`, and the subdirectories were the same as they are for the node.  I know a planned use case for this tool is to clone main-net for dry-run testing of upgrades...

Reply:
This was a migration to load blocks from their old files into storage, and can be removed now.